src/ui/public/paginated_table/paginated_table.html

```diff
@@ -15,6 +15,7 @@
             <kbn-info ng-if="col.info" info="{{ col.info }}" placement="top"></kbn-info>
             <i
               ng-if="col.sortable !== false"
+              tabindex="0"
```

Comment:
`tabindex` needs to be on the element, that also allows the click (`ng-click`). Also this only makes it focusable, but not "keyboard accessible", that the user can activate the action by keyboard. You would need to add the `kbnAccessibleClick` directive too.